torch/onnx/symbolic_opset9.py

```diff
@@ -746,6 +746,12 @@ def atan2(g: jit_utils.GraphContext, self, other):
 @symbolic_helper.quantized_args(True, scale=1.0 / 256.0, zero_point=0)
 @_beartype.beartype
 def sigmoid(g: jit_utils.GraphContext, self):
+    """Applies the sigmoid function element-wise. The sigmoid function is defined as: sigmoid(x) = 1 / (1 + exp(-x)).
```

Comment:
Similar to pr #128055, this function convert function torch.sigmoid into onnx operator. It is never called directly by the user.